### PR TITLE
obtain user info from id_token

### PR DESCRIPTION
**Pull request by YichingChiang**
* obtain user information from id_token instead calling Profile API.
* remove status_message from Profile class and view.
* update Spring Boot.

### CODE REVIEW

File: src/main/java/com/linecorp/sample/login/infra/line/api/v2/response/IdToken.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    }
+
+    public String getDispalyName() {
+        return name;
+    }
+
+    public String getPictureUrl() {
+        return picture;
+    }
+}
```
Comment (KosukeKamiya) [inline src/main/java/com/linecorp/sample/login/infra/line/api/v2/response/IdToken.java]: The "no newline at end of file" error at this point.

File: src/main/java/com/linecorp/sample/login/infra/utils/JwtUtils.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+     */
+    public IdToken decode(String id_token) {
+        try {
+            DecodedJWT jwt = JWT.decode(id_token);
+            return new IdToken(jwt);
+        } catch (JWTDecodeException e) {
+            throw new RuntimeException(e);
+        }
+    }
+}
```
Comment (KosukeKamiya) [inline src/main/java/com/linecorp/sample/login/infra/utils/JwtUtils.java]: The "no newline at end of file" error at this point.

File: src/main/java/com/linecorp/sample/login/infra/line/api/v2/response/IdToken.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.sample.login.infra.line.api.v2.response;
+
+import com.auth0.jwt.interfaces.DecodedJWT;
+
+public final class IdToken {
+    private DecodedJWT jwt;
```
Comment (cosmic-cowboy) [inline src/main/java/com/linecorp/sample/login/infra/line/api/v2/response/IdToken.java]: @YichingChiang 
I think this class doesn't need DecodedJWT class.
DecodedJWT is not IdToken's field.
https://developers.line.me/ja/docs/line-login/web/integrate-line-login/#part-715703e07a7d3b32
So, Could you remove this field ?
And you can set constructor's argument to decoded value.

**Comment by cosmic-cowboy** [comment]
LGTM

**Comment by cosmic-cowboy** [comment]
@

**Comment by cosmic-cowboy** [comment]
@YichingChiang 
I overlooked Profile's object.
If you never get Profile's object for response object from LINE, could you remove this object.
https://github.com/YichingChiang/line-login-starter/blob/master/src/main/java/com/linecorp/sample/login/infra/line/api/v2/response/Profile.java

**Comment by cosmic-cowboy** [comment]
@
LGTM

**Pull request merged**